addons/binding/org.openhab.binding.gpio/build.properties

```diff
@@ -0,0 +1,7 @@
+source.. = src/main/java/
```

Comment:
A default build.properties should look like:

```
source.. = src/main/java/
output.. = target/classes
bin.includes = META-INF/,\
               .,\
               OSGI-INF/,\
               ESH-INF/,\
               about.html

```

Note that your about.html is also missing, see [coding guidelines B.2](http://docs.openhab.org/developers/development/guidelines.html#b-osgi-bundles)